lib/puppet/application/ssl.rb

```diff
@@ -45,6 +45,14 @@ def help
 ACTIONS
 -------
 
+* bootstrap:
+  Perform all of the steps necessary to request and download a client
+  certificate. If autosigning is disabled, then puppet will wait every
+  `waitforcert` seconds for its certificate to be signed. To prevent
+  puppet from waiting, specify a time of 0. Since `waitforcert` is a
```

Comment:
Does "prevent puppet from waiting" mean that it will only attempt once, or that there is no wait time between attempts?

Reply:
Looks like from the commit message the answer is "only attempt once", but that's not clear from this documentation.

Reply:
it will only attempt once, but never wait. I tried to match the verbiage in the agent application https://github.com/puppetlabs/puppet/blob/master/lib/puppet/application/agent.rb#L297-L305

Reply:
How about "To only attempt once and never wait, specify a time of 0"?

Reply:
Yeah, that's a lot clearer. 👍

Reply:
Done